Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Learn about building .NET container images:
+# https://github.com/dotnet/dotnet-docker/blob/main/samples/README.md
+FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build
+WORKDIR /source
+
+# copy csproj and restore as distinct layers
+COPY *.sln .
+COPY nuget.config .
+COPY HerPublicWebsite/*.csproj HerPublicWebsite/
+COPY HerPublicWebsite.BusinessLogic/*.csproj HerPublicWebsite.BusinessLogic/
+COPY HerPublicWebsite.Data/*.csproj HerPublicWebsite.Data/
+COPY Lib/ Lib/
+RUN dotnet restore HerPublicWebsite/ --use-current-runtime
```

Comment:
Slightly modified from the .NET example, so that it also copies our custom Lib and nuget config